e2e/playwright/support/components/jira_setup_flow.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Page} from '@playwright/test';
+
+import {ChannelsPage} from '@e2e-support/ui/pages';
+
+import SetupFlow from './base/setup_flow';
+
+export default class JiraSetupFlow extends SetupFlow {
```

Comment:
@trilopin I'm curious what you think about using subclasses to share certain functionality among test components like this